README.md

```diff
@@ -79,9 +79,10 @@ This will prompt for parameters for initialization. Some of these parameters are
  * ``input_cloud``: Cloud provider you use with Databricks (AWS, Azure, or GCP).
 
 Others must be correctly specified for CI/CD to work:
- * ``input_cicd_platform`` : CI/CD platform of choice (GitHub Actions or GitHub Actions for GitHub Enterprise Servers or Azure DevOps)
- * ``input_databricks_staging_workspace_host``: URL of staging Databricks workspace, used to run CI tests on PRs and preview config changes before they're deployed to production.
-   We encourage granting data scientists working on the current ML project non-admin (read) access to this workspace,
+ * ``input_cicd_platform`` : CI/CD platform of choice (GitHub Actions or GitHub Actions for GitHub Enterprise Servers or Azure DevOps or Gitlab)
+ * ``input_databricks_test_workspace_host``: URL of staging Databricks workspace, used to run CI tests on PRs before they're deployed to staging.
```

Comment:
Do we need this? `input_databricks_test_workspace_host` is for dev or staging? If it's the same URL as staging we don't need another parameter right?

Reply:
I have been working and developing this PR in a scenario with 4 environments (dev, integration test, staging, prod).

The parameter gives you the option to have a separate integration test environment, while you can still choose to use the same workspace as in staging, in case you do not have/need a separate workspace.

Reply:
NOTE that I am using the integration test environment in the gitlab pipelines only.
so we either need to:
- modify Github/Azure pipelines to use it
- make this parameter restricted to the gitlab option only

Reply:
Got it, generally we try to minimize the number of parameters so users don't get overwhelmed with the number of knobs, and it seems that this parameter is only used in one spot (`databricks.yml`). I think it would be better to add docs on how to change this if they want, but not add an extra parameter just for this one spot especially if in most cases it would stay the same as staging

Reply:
removed the parameter